diff --git a/src/MetadataReader.php b/src/MetadataReader.php
index 506099b..8edb0d9 100755
--- a/src/MetadataReader.php
+++ b/src/MetadataReader.php
@@ -14,7 +14,7 @@ class MetadataReader{
 
 		foreach ($entityDescriptors as $entityDescriptor) {
 			$idpSSODescriptor = Utilities::xpQuery($entityDescriptor, './saml_metadata:IDPSSODescriptor');
-			
+
 			if(isset($idpSSODescriptor) && !empty($idpSSODescriptor)){
 				array_push($this->identityProviders,new IdentityProviders($entityDescriptor));
 			}
